<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Put links here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dwnBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Store for later</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linkBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cleanChk">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>smart download</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>unused_now</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Links Loaded:</string>
    </property>
   </widget>
   <widget class="QLabel" name="linkNum">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>62</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="directoryInput">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>~/Downloads</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>62</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
